### PR TITLE
Feat/checkbox ts2

### PR DESCRIPTION
**Pull request by jessiehuff**
**What**: Fixes #1759

**Additional issues**:

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.test.tsx [hunk 1]
```diff
@@ -1,8 +1,6 @@
-
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import Avatar from './Avatar';
-
+import { Avatar } from './Avatar';
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Avatar/Avatar.test.tsx]: Also change Avatar.tsx to not have a default export.

**Comment by redallen** [comment]
The checkboxes are broken in the preview: https://1762-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/checkbox/

File: packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  label = undefined,
+  'aria-label': ariaLabel = '',
+  ...props
+}) => (
+  <div className={css(styles.check, className)}>
+    <input
+      {...props}
+      className={css(styles.checkInput)}
+      type="checkbox"
+      onChange={(event: React.FormEvent<HTMLInputElement>) => {
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.tsx]: Try `onChange={(event: React.FormEvent<HTMLInputElement>) => this.props.onChange(event.currentTarget.checked, event)}`

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -12,13 +12,6 @@ export interface AvatarProps
...
-  alt,
-  ...props
-}) => (
-    <img {...props} src={src} alt={alt} className={css(styles.avatar, className)} />
-  );
-
-export default Avatar;
+export const Avatar: React.FunctionComponent<AvatarProps> = ({ className = '', src = '', alt, ...props }) => (
+  <img {...props} src={src} alt={alt} className={css(styles.avatar, className)} />
+);
```
Comment (dlabaj) [inline packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx]: export default Avatar is needed as well.

File: packages/patternfly-4/react-core/src/components/Avatar/index.ts [hunk 4]
```diff
@@ -1 +1 @@
-export { default as Avatar } from './Avatar';
+export { Avatar } from './Avatar';
```
Comment (dlabaj) [inline packages/patternfly-4/react-core/src/components/Avatar/index.ts]: Export * from './Avatar'

**Comment by nicolethoen** [comment]
@jessiehuff can you also add integration tests for this component as outlined on this [README](https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-integration)?

**Pull request merged**